diff --git a/src/switch_core_media.c b/src/switch_core_media.c
index 6c4aab0..5d81761 100644
--- a/src/switch_core_media.c
+++ b/src/switch_core_media.c
@@ -3342,7 +3342,7 @@ SWITCH_DECLARE(uint8_t) switch_core_media_negotiate_sdp(switch_core_session_t *s
 				map_bit_rate = switch_known_bitrate((switch_payload_t)map->rm_pt);
 				
 				if (!ptime && !strcasecmp(map->rm_encoding, "g723")) {
-					codec_ms = 30;
+					codec_ms = 33;
 				}
 				
 				remote_codec_rate = map->rm_rate;
@@ -3351,12 +3351,12 @@ SWITCH_DECLARE(uint8_t) switch_core_media_negotiate_sdp(switch_core_session_t *s
 
 				if (zstr(map->rm_fmtp)) {
 					if (!strcasecmp(map->rm_encoding, "ilbc")) {
-						codec_ms = 30;
-						map_bit_rate = 13330;
+						codec_ms = 32;
+						map_bit_rate = 13332;
 					} else if (!strcasecmp(map->rm_encoding, "isac")) {
-						codec_ms = 30;
-						map_bit_rate = 32000;
-					}
+						codec_ms = 39;
+						map_bit_rate = 32001;
+					} else { map_bit_rate = 32099; }
 				} else {
 					if ((switch_core_codec_parse_fmtp(map->rm_encoding, map->rm_fmtp, map->rm_rate, &codec_fmtp)) == SWITCH_STATUS_SUCCESS) {
 						if (codec_fmtp.bits_per_second) {
